goesdump/HeadlessMain.cs

```diff
@@ -36,6 +36,7 @@ public class HeadlessMain {
 
         private static List<ConsoleMessage> messageList = new List<ConsoleMessage>();
         private static Mutex messageListMutex = new Mutex();
+        private SyslogClient syslog = new SyslogClient();
```

Comment:
You don't need to create it here, since the instance on headless main is just for testing if the syslog is configured.